### PR TITLE
Remove github.com/alexflint/go-arg

### PR DESCRIPTION
**Pull request by jameshalsall**

### CODE REVIEW

File: config/helper.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	var i []int
+	for _, pt := range strings.Split(envVal, ",") {
+		val, err := strconv.Atoi(pt)
+		if err != nil {
+			return nil, err
+		}
+		i = append(i, val)
+	}
+	return i, nil
+}
```
Comment (hgajjar) [inline config/helper.go:39]: could you add some tests for this helper methods please?

Comment (jameshalsall) [reply config/helper.go:39]: they are covered by the config tests already?

File: config/helper.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+func envVarAsIntSlice(name string) ([]int, error) {
+	envVal := os.Getenv(name)
+	if envVal == "" {
+		return []int{}, nil
+	}
+
+	var i []int
+	for _, pt := range strings.Split(envVal, ",") {
+		val, err := strconv.Atoi(pt)
```
Comment (hgajjar) [inline config/helper.go:32]: does it work ok if there is a space? ex. `KAFKA_RETRY_INTERVALS=1, 5`

**Review by hgajjar — APPROVED** [review]

File: config/helper.go (hunk 2 above)
Comment (jameshalsall) [reply config/helper.go:32]: will handle that, and add tests to cover those edge cases directly, you are right
